otterdog/providers/github/rest/org_client.py

```diff
@@ -30,6 +30,15 @@ async def get_id(self, org_id: str) -> int:
         except GitHubException as ex:
             raise RuntimeError(f"failed retrieving id for org '{org_id}':\n{ex}") from ex
 
+    async def is_archived(self, org_id: str) -> bool:
```

Comment:
that code is not used but I  guess you added it for checking whether an org is archived?

Reply:
My bad, this commit should go in the other PR. I should not do interactive rebase and reordering of commits late at night :) 

I 'll push a change